cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -148,12 +148,23 @@ function processEvent(event, callback) {
 
   let messageText;
 
-  if (message.source == "aws.config") {
+  if (message.source === "aws.config") {
     const alarmName = message.detail.configRuleName;
     const newState = message.detail.newEvaluationResult.complianceType;
     messageText = `${alarmName} state is now ${newState}`;
     sendMessageToSlack(messageText, callback);
-  } else if (message.source == "aws.guardduty") {
+  } else if (message.source === "aws.trustedadvisor") {
+    const detail_type = message["detail-type"];
```

Comment:
Not worth changing, and we should just have this in the linter to enforce it, but we should be using camel case, not snake case, for variable names in JavaScript.

Reply:
fixed